gemrb/core/Map.cpp

```diff
@@ -2439,7 +2397,15 @@ bool Map::AdjustPositionY(Point &goal, unsigned int radiusx,  unsigned int radiu
 	return false;
 }
 
-void Map::AdjustPosition(Point &goal, unsigned int radiusx, unsigned int radiusy)
+void Map::AdjustPositionSearchmap(Point &goal, unsigned int radiusx, unsigned int radiusy)
+{
+	NavmapPoint nmptGoal(goal.x * 16 + 8, goal.y * 12 + 6);
```

Comment:
Codacy found an issue: [Variable 'nmptGoal' is assigned a value that is never used.](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=3427588)